### PR TITLE
Document network section

### PR DESCRIPTION
**Pull request by teclator**
Document the `network` section

- https://trello.com/c/Jao9dDKQ/2477-5-make-forwarding-configurable-for-installer
- https://bugzilla.suse.com/show_bug.cgi?id=1186280

### CODE REVIEW

File: doc/control-file.md [hunk 1]
```diff
@@ -1224,6 +1234,25 @@ This is the full list of SLE-15 SP2:
 </subvolumes>
 ```
 
+## Network
+
+The network sections contains some options that allows to modify network
```
Comment (imobachgs) [inline doc/control-file.md:1239]: "some options that allow" or simply "some options to modify"

File: doc/control-file.md [hunk 2]
```diff
@@ -1224,6 +1234,25 @@ This is the full list of SLE-15 SP2:
 </subvolumes>
 ```
 
+## Network
+
+The network sections contains some options that allows to modify network 
+configuration defaults.
+
+  * `force_static_ip` *(boolean, default: `false`)*
+    New connections will be initialized using an static bootproto.
```
Comment (imobachgs) [inline doc/control-file.md:1243]: What about "Initialize new connections using a static bootproto"?

File: doc/control-file.md [hunk 3]
```diff
@@ -1224,6 +1234,25 @@ This is the full list of SLE-15 SP2:
...
+  * `startmode` *(string)*
+    When to bring up an interface. Possible values are `hotplug`, `auto`, `ifplugd`, 
+    `manual`, `nfsroot` or `off`.
+  * `network_manager`
+    Whether NetworkManager should be the default backend to be used or not.
```
Comment (imobachgs) [inline doc/control-file.md:1248]: "Whether NetworkManager should be the default backend."

**Review by imobachgs — COMMENTED** [review]
Some minor comments. Otherwise, it looks good.

File: doc/control-file.md (hunk 2 above)
Comment (teclator) [reply doc/control-file.md:1243]: I will adapt with the suggestion but was event tempted to remove it completely as currently the real behavior is only about "showing an extra" paragraph as part of the help text. But the option is still there.

**Review by imobachgs — APPROVED** [review]

**Pull request merged**